src/main/java/com/entropyteam/entropay/employees/services/ReportService.java

```diff
@@ -291,6 +290,33 @@ private List<PtoReportClientDto> getPtoReportClientDtos(List<Pto> ptoList, Integ
         }).filter(Objects::nonNull).collect(Collectors.toList());
     }
 
+    public Page<PtoReportDetailDto> getPtoReportAllDetails(ReactAdminParams params) {
```

Comment:
I think this method shouldn't receive an employee id or client Id. It should export all employees or all clients